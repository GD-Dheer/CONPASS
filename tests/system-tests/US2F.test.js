import React from 'react';
import Enzyme, { mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ShuttleSchedule from '../../components/home/menu/shuttleBusSchedule';
import SearchBar from '../../components/searchBar';

Enzyme.configure({ adapter: new Adapter() });

it('System Test: US2F', () => {
  const wrapper = mount(<ShuttleSchedule />);
  // const sgwCampusData = wrapper.children(React.ScrollView).children(React.SectionList).sections;
});

describe('SearchBar component', () => {
  it('should render correctly in "debug" mode', () => {
    const searchBarcomponent = shallow(<SearchBar debug />);
    expect(searchBarcomponent).toMatchSnapshot();
  });
});
