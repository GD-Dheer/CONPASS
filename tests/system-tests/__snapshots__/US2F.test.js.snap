// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar component should render correctly in "debug" mode 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "borderRadius": 10,
      "position": "absolute",
      "top": 25,
      "width": "90%",
      "zIndex": 201,
    }
  }
>
  <View>
    <ForwardRef(Themed.Tooltip)
      backgroundColor="#b5e3e6"
      height={100}
      popover={
        <Text>
          Include
          <Text
            style={
              Object {
                "fontWeight": "bold",
              }
            }
          >
             "near LOY"
          </Text>
           
          or
          <Text
            style={
              Object {
                "fontWeight": "bold",
              }
            }
          >
             "near SGW"
          </Text>
           
          to find places around campus!
        </Text>
      }
    >
      <ForwardRef(Themed.SearchBar)
        autoCorrect={false}
        blurOnSubmit={true}
        containerStyle={
          Object {
            "borderRadius": 10,
            "borderWidth": 1,
            "height": 45,
            "justifyContent": "center",
          }
        }
        lightTheme={true}
        onBlur={[Function]}
        onChangeText={[Function]}
        onClear={[Function]}
        onSubmitEditing={[Function]}
        onTouchStart={[Function]}
        padding={5}
        placeholder="Search..."
        platform="android"
        returnKeyType="search"
        searchIcon={<Icon />}
        value=""
      />
    </ForwardRef(Themed.Tooltip)>
  </View>
</View>
`;
